api/v1/checks.go

```diff
@@ -196,6 +196,44 @@ func (c ResticCheck) GetType() string {
 	return "restic"
 }
 
+type JmeterCheck struct {
+	// JmxFrom defines tge ConfigMap or Secret reference to get the JMX test plan
+	JmxFrom kommons.EnvVar `yaml:"jmxFrom" json:"jmxFrom"`
```

Comment:
```suggestion
	Jmx kommons.EnvVar `yaml:"jmxFrom" json:"jmx"`
```

If EnvVar has the valueFrom then it shouldn't be duplicted in the var name

Reply:
ack